device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/gearbox_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "sesto-1",
+      "address": "0x1000",
+      "lib_name": "libsai_phy_sesto-1.so",
+      "firmware_path": "/tmp/phy-sesto-1.bin",
+      "config_file": "/usr/share/sonic/hwsku/sesto-1.json",
+      "sai_init_config_file": "/usr/share/sonic/hwsku/sesto-1.bcm",
```

Comment:
where is this sai_init_config_file checked in? I think we need this file to be checked in.

Reply:
This initial gearbox delivery is targeting VS and therefore doesn't use any .bcm file. Please note that this is currently a placeholder and can change as required.